src/app/business/common/components/detail/components/metadata.js

```diff
@@ -160,6 +160,38 @@ OwnerMetadata.defaultProps = {
     owner: '',
 };
 
+export const MetadataMetadata = ({metadata}) => {
+    const keys = Object.keys(metadata);
+
+    const li = css`
+        font-family: SFMono-Regular,Consolas,Liberation Mono,Menlo,Courier,monospace
```

Comment:
There is a `monospaceFamily` variable available that you should use instead.

Reply:
Indeed, thanks!